js/components/wysiwyg.js

```diff
@@ -185,6 +185,18 @@ Fliplet.FormBuilder.field('wysiwyg', {
 
         editor.on('keydown', $vm.addBulletedListShortcutsMac);
 
+        editor.on('focus', function() {
+          var $el = $(editor.iframeElement);
+
+          $el.parents().eq(1).addClass('focus-outline');
```

Comment:
```suggestion
          $el.parent().parent().addClass('focus-outline');
```

![image](https://user-images.githubusercontent.com/290733/109224588-12504080-77b4-11eb-83d8-00cca4319dbc.png)